modules/monitoring-creating-user-defined-workload-monitoring-configmap.adoc

```diff
@@ -6,7 +6,7 @@
 [id="creating-user-defined-workload-monitoring-configmap_{context}"]
 = Creating a user-defined workload monitoring config map
 
-To configure the components that monitor user-defined projects, you must create the `user-workload-monitoring-config` `ConfigMap` object in the `openshift-user-workload-monitoring` project.
+You can configure the Cluster Monitoring Operator (CMO) by creating the `user-workload-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project. The CMO then configures the components that monitor user-defined projects.
```

Comment:
```suggestion
You can configure the user workload monitoring components by creating the `user-workload-monitoring-config` `ConfigMap` object in the `openshift-user-workload-monitoring` project. 
```

Reply:
@juzhao 
Should we not specify here that it is also done by the CMO?
```suggestion
You can configure the user workload monitoring components by creating the `user-workload-monitoring-config` `ConfigMap` object in the `openshift-user-workload-monitoring` project. The Cluster Monitoring Operator (CMO) then configures the components that monitor user-defined projects.
```

Reply:
Just saw you edited comment :smile: :+1: